lib/chromic_pdf/supervisor.ex

```diff
@@ -244,6 +244,7 @@ defmodule ChromicPDF.Supervisor do
       @type global_option ::
               {:offline, boolean()}
               | {:disable_scripts, boolean()}
+              | {:raise_on_unhandled_runtime_exceptions, boolean()}
```

Comment:
to me, still this would be the better default, adding `ignore_js_errors: true` as an option instead, to explicitly ignore things...

Reply:
But assuming we may have users with templates with unhandled exceptions out there (printing may still work just fine), this would be a breaking change... idk

I'd instead consider warning about them unconditionally (so regardless of this option), wdyt?

Reply:
Warning sounds good. Maybe we can still have the option to silence those warnings if they appear explicitly (because JS...)

Reply:
I just guess not ignoring JS issues by default completely without output will save us some issue work in the long run.

Reply:
^^ hence I stopped implementing it when I realized that. Solutions would be:

1) somehow store the global options and pass them down to the print job
2) make this option a print job option (i.e. per `print_to_pdf` call), although IMO it feels more like a global option